testutil/validatormock/validatormock.go

```diff
@@ -338,6 +338,43 @@ func ProposeBlindedBlock(ctx context.Context, eth2Cl Eth2Provider, signFunc Sign
 	return eth2Cl.SubmitBlindedBeaconBlock(ctx, signedBlock)
 }
 
+// this is register (singular) but the underlying submit function is registrations should it be changed to handle multiple registrations
```

Comment:
this is register (singular) but the underlying submit function is registrations should it be changed to handle multiple registrations